shared/team-building/container.tsx

```diff
@@ -282,6 +283,59 @@ const deriveRolePickerArrowKeyFns = memoize(
   })
 )
 
+const alphabet = 'abcdefghijklmnopqrstuvwxyz'
+const isAlpha = (letter: string) => alphabet.includes(letter)
+const letterToAlphaIndex = (letter: string) => alphabet.indexOf(letter)
+
+// Returns array with 28 entries
+// 0 - "Recommendations" section
+// 1-26 - a-z sections
+// 27 - 0-9 section
+const sortAndSplitRecommendations = memoize(
+  (results: Unpacked<typeof deriveSearchResults>): Array<SearchRecSection> | null => {
+    if (!results) return results
```

Comment:
I think this would be clearer if it explicitly returned null.
```suggestion
    if (!results) return null
```